engine/src/main/java/org/terasology/rendering/nui/layouts/RowLayout.java

```diff
@@ -34,31 +34,60 @@
 import java.util.Map;
 
 /**
+ * A layout that places widgets side-by-side, with support for relative widths
```

Comment:
I guess maybe 'in a row' would be better phrasing, since there might be spacing between the widgets.